packages/remix-dev/compiler/manifest.ts

```diff
@@ -93,9 +93,24 @@ export async function create({
   invariant(entry, `Missing output for entry point`);
 
   optimizeRoutes(routes, entry.imports);
-  let version = getHash(JSON.stringify({ entry, routes })).slice(0, 8);
 
-  return { version, entry, routes, cssBundleHref, hmr };
+  let fingerprintedValues = {
```

Comment:
I've refactored the code here to help people avoid the mistake of adding a new value to the manifest but forgetting to include it in the hash.